File: meutils/src/main/java/me/hp/meutils/ui/mvp/IView.java
package me.hp.meutils.ui.mvp;

import androidx.lifecycle.LifecycleOwner;

/**
 * @author: hepingdev
 * @created: 2019/1/19.
 * @desc:
 */
public interface IView extends LifecycleOwner {

    default void showLoadDialog(){}

    default void hideLoadDialog(){}

}

File: meutils/src/main/java/me/hp/meutils/ui/mvp/IView.kt
package me.hp.meutils.ui.mvp

import androidx.lifecycle.LifecycleOwner

/**
 * @author: HePing
 * @created: 2020/1/1.
 * @desc:
 */
interface IView : LifecycleOwner {

    fun showLoadDialog()

    fun hideLoadDialog()

}
